### PR TITLE
Installation process for GFortran on OpenBSD

### PR DESCRIPTION
**Pull request by brocolis**
Installation process for GFortran on OpenBSD.

### CODE REVIEW

**Review by LKedward — APPROVED** [review]
Thanks Carlos @brocolis - I don't have any experience with OpenBSD, but these changes look good to me.

**Comment by brocolis** [comment]
Thanks for reviewing it, @LKedward. I have one day of experience with OpenBSD 😸. Today I discovered that we can install GFortran with `pkg_add g95`, no need to include full version number. I'll update the PR.

File: learn/os_setup/install_gfortran.md [hunk 1]
```diff
@@ -95,6 +95,17 @@ Install a gcc version:
...
 ```
+
+## OpenBSD
+```bash
+pkg_add g95
+```
+
+On OpenBSD, the GFortran executable is named `egfortran`. To test it, type:
+```bash
+egfortran -v
```
Comment (awvwgk) [inline learn/os_setup/install_gfortran.md:106]: Is this an issue for fpm to discover the right compiler? It probably requires to set `--compiler egfortran` on every as well.

Comment (brocolis) [reply learn/os_setup/install_gfortran.md:106]: @awvwgk, yes, `fpm` works with `--compiler egfortran`.  `export FPM_COMPILER=egfortran` also works 👍.

**Review by awvwgk — APPROVED** [review]

**Comment by milancurcic** [comment]
Very nice, thanks @brocolis!

**Pull request merged**